mpd/__init__.py

```diff
@@ -0,0 +1,215 @@
+import os
```

Comment:
plase add a docstring containing a description and a synopsis. synopsis is important since else nobody can use the extension.

Reply:
Sure!